Alcatraz/Helpers/ATZStyleKit.m

```diff
@@ -99,16 +96,37 @@ + (void)drawText:(NSString*)text withColor:(NSColor *)color centeredInRect:(NSRe
     CGContextAddPath(context, clippingPath);
     CGContextClip(context);
 
+    // Text font & style
+    NSFont *font = [NSFont fontWithName:@"HelveticaNeue-Light" size:12];
```

Comment:
Oh. Ok, fixed in [13a20cd](https://github.com/guillaume-algis/Alcatraz/commit/13a20cd943181bca9069f51bd1b5f1dfb6af40c2)